app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -50,7 +52,7 @@ class MainFragment : Fragment() {
     private val MY_REQUEST_CODE = 111
     val notificationManager =
         context?.getSystemService(NOTIFICATION_SERVICE) as NotificationManager?
-
+    val audioManager by lazy { activity?.getSystemService(Context.AUDIO_SERVICE) as AudioManager }
```

Comment:
instead of activity use requireActivity to avoid null check.

Reply:
@rob729 Ok